core/test/functional/api/v0.1/subscribers_spec.js

```diff
@@ -229,7 +229,7 @@ describe('Subscribers API', function () {
 
                 jsonResponse.meta.stats.imported.should.equal(2);
                 jsonResponse.meta.stats.duplicates.should.equal(0);
-                jsonResponse.meta.stats.invalid.should.equal(1);    // TODO: should return 0
+                jsonResponse.meta.stats.invalid.should.equal(0);    // TODO: should return 0
```

Comment:
```suggestion
                jsonResponse.meta.stats.invalid.should.equal(0);
```
no need for the comment as the assertion is being fixed :)

Reply:
Done 👍